Puzzlepart/did
# Upgrading simple-oauth2

### Your checklist for this pull request

### Review checklist

### Description
Upgrading [simple-oauth2](https://www.npmjs.com/package/simple-oauth2) from 2.4.0 to 4.1.0

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Verify you can log on to did

### Related issues
Closes #264
